settings/settings.go

```diff
@@ -1,50 +1,22 @@
 package settings
 
 import (
-	"os"
-	"time"
+	"fmt"
 
 	"github.com/joho/godotenv"
 )
 
-func InitSettings() {
-	godotenv.Load(".env")
-
-	Debug = os.Getenv("DEBUG") == "true"
-	if Debug {
-		EMAIL_INTERVAL = 1 * time.Minute
+func InitSettings() error {
+	err := godotenv.Load(".env")
+	if err != nil {
+		return fmt.Errorf("No .env file found: %w", err)
```

Comment:
https://google.github.io/styleguide/go/decisions.html#error-strings

Error strings should not be capitalized (unless beginning with an exported name, a proper noun or an acronym) and should not end with punctuation. This is because error strings usually appear within other context before being printed to the user.